lib/src/text_controller.dart

```diff
@@ -0,0 +1,78 @@
+part of 'hooks.dart';
+
+class _TextEditingControllerHookCreator {
+  const _TextEditingControllerHookCreator();
+
+  /// Creates a [TextEditingController] that will be disposed automatically.
+  ///
+  /// The [text] parameter can be used to set the initial value of the
+  /// controller.
```

Comment:
It should mention that updating `text` has no impact whatsoever.
Bonus point if you include a snippet to show how to update it (with something like `useEffect` for example)

Reply:
I've added this and an example on how to have the controller reflect changes from another `ValueListenable` to the `useTextEditingController` field, as it affects both the text and the value.